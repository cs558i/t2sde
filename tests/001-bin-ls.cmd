#!/bin/sh

[ "$QEMU" ] || exit 43

[ -e $SYSROOT/bin/ls ] || exit 42

$QEMU -chroot $SYSROOT bin/ls -d etc proc dev sys tmp
