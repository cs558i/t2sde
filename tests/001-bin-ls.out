dev
etc
proc
sys
tmp
